### ISSUE TITLE
aws_s3_bucket  policy not accurate

### ISSUE THREAD

**Issue by tsenay** [JamesWoolfenden/pike#75]
**Describe the bug**
When creating policy for an aws_s3-bucket, the output is not accurate

**To Reproduce**

Using `pike version v0.3.26`

1. Use the following terraform source

```hcl
resource "aws_s3_bucket" "test" {
  bucket = "test"
  tags = {
    tag_1 = "value_1"
  }
}
```

2. run `pike scan -o json > policy.json`

**Expected behavior**
the following policy is generated
```json
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "VisualEditor0",
            "Effect": "Allow",
            "Action": [
                "s3:CreateBucket",
                "s3:DeleteBucket",
                "s3:GetAccelerateConfiguration",
                "s3:GetBucketAcl",
                "s3:GetBucketCORS",
                "s3:GetBucketLogging",
                "s3:GetBucketObjectLockConfiguration",
                "s3:GetBucketPolicy",
                "s3:GetBucketRequestPayment",
                "s3:GetBucketTagging",
                "s3:GetBucketVersioning",
                "s3:GetBucketWebsite",
                "s3:GetEncryptionConfiguration",
                "s3:GetLifecycleConfiguration",
                "s3:GetObject",
                "s3:GetObjectAcl",
                "s3:GetReplicationConfiguration",
                "s3:ListBucket"
            ],
            "Resource": [
                "*"
            ]
        }
    ]
}


```

the `s3:PutBucketTagging` api should be in statement actions

**Platform**

- HW: Apple Silicon M1 Pro
- OS: macOS 14.7.1

**Comment by JamesWoolfenden** [comment]
thanks for logging this. will look at asap

**Comment by JamesWoolfenden** [comment]
fixed in v0.3.28 fat fingers typo!
